src/app/calculator/process-cooling/cooling-tower/cooling-tower-form/cooling-tower-form.component.html

```diff
@@ -1,218 +1,223 @@
+          <span *ngIf="form.controls.flowRate.errors.required">Value required.</span>
         </span>
-      </label>
-  
-      <div class="form-section pt-2" *ngIf="isBaseline">
-        <div class="form-group">
-          <label class="small" for="{{'operationalHours_'+idString}}">Annual Operating Hours</label>
-          <div class="input-group">
-            <span class="input-group-addon op-hour-addon units">
-              <a class="click-link" (click)="openOperatingHoursModal()"><span class="fa fa-calculator"></span></a>
+      </div>
+
+      <div class="form-group" *ngIf="!form.controls.userDefinedCoolingLoad.value">
+        <label for="temperatureDifference">Temperature Difference</label>
+        <div class="input-group">
+          <input class="form-control" name="temperatureDifference" id="temperatureDifference"
+            formControlName="temperatureDifference" type="number" (input)="calculate()"
+            (focus)="focusField('temperatureDifference')" onfocus="this.select();">
+          <span *ngIf="settings.unitsOfMeasure != 'Metric'" class="input-group-addon units">&#8457;</span>
+          <span *ngIf="settings.unitsOfMeasure == 'Metric'" class="input-group-addon units">&#8451;</span>
+          <span class="alert-danger pull-right small"
+            *ngIf="form.controls.temperatureDifference.invalid && !form.controls.temperatureDifference.pristine">
+            <span *ngIf="form.controls.temperatureDifference.errors.greaterThan == 0">Value must be greater than 0.
+              <span *ngIf="settings.unitsOfMeasure != 'Metric'">&#8457;</span>
+              <span *ngIf="settings.unitsOfMeasure == 'Metric'">&#8451;</span>.
             </span>
-            <input class="form-control" name="operationalHours" id="{{'operationalHours_'+idString}}"
-              formControlName="operationalHours" type="number" (input)="calculate()" (focus)="focusField('operationalHours')" onfocus="this.select();">
-            <span class="units input-group-addon">hrs/yr</span>
-          </div>
+          </span>
+        </div>
+      </div>
+      <div class="form-group pt-1">
+        <label>Cooling Load
+          <a class="click-link small form-text" (click)="showHideInputField()"
+            *ngIf="form.controls.userDefinedCoolingLoad.value">Calculate Cooling Load</a>
+          <a class="click-link small form-text" (click)="showHideInputField()"
+            *ngIf="!form.controls.userDefinedCoolingLoad.value">Define Cooling Load</a>
+        </label>
+        <div class="input-group" *ngIf="form.controls.userDefinedCoolingLoad.value == true">
+          <input class="form-control" name="coolingLoad" id="coolingLoad" formControlName="coolingLoad" type="number"
+            (input)="calculate()" (focus)="focusField('coolingLoad')" onfocus="this.select();">
+          <span class="units input-group-addon" *ngIf="settings.unitsOfMeasure != 'Metric'">MMBtu/h</span>
```

Comment:
These units correct? `MMBtu/h` and `GJ` ? They may be but smells wrong.

Reply:
Verified GJ by @koay9f . Adding "/h"